resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/driver/Dockerfile

```diff
@@ -17,5 +17,14 @@ ENV SPARK_HOME /opt/spark
 
 WORKDIR /opt/spark
 
-# This class will also require setting a secret via the SPARK_APP_SECRET environment variable
-CMD exec bin/spark-class org.apache.spark.deploy.rest.kubernetes.KubernetesSparkRestServer --hostname $HOSTNAME --port $SPARK_DRIVER_LAUNCHER_SERVER_PORT --secret-file $SPARK_SUBMISSION_SECRET_LOCATION
+CMD SSL_ARGS="" && \
```

Comment:
This is tricky - I think using `SPARK_DAEMON_JAVA_OPTS` is another option here with passing system properties through -D. But I like that here it's clear from the Docker File how the driver rest server is to be configured.